superchain/configs/sepolia/semver.yaml

```diff
@@ -3,7 +3,7 @@ l1_erc721_bridge: 2.1.0
 l1_standard_bridge: 2.1.0
 l2_output_oracle: 1.8.0
 optimism_mintable_erc20_factory: 1.9.0
-optimism_portal: 2.5.0
+optimism_portal: 3.3.0
```

Comment:
Where do I sanity check this?

Reply:
Good question. I think at the moment we are just trusting whichever version is deployed on OP Sepolia. You can sanity check the version here https://sepolia.etherscan.io/address/0x16Fc5058F25648194471939df75CF27A2fdC48BC#readProxyContract